.travis.yml

```diff
@@ -7,3 +7,7 @@ addons:
   postgresql: "9.4"
 before_script:
   - mix ecto.create
+script:
+  - mix test
```

Comment:
Actually I was unaware that it did it by default till you did, most of the project I have seen using travis explicitly do this including phoenix. 

I'm ok with trying it without it but now having the after_script it doesn't quite read like it's doing the tests between unless you just happen to know that it does that by default. I'd imagine the default then is that it as if the script: tag was there and after_script will run after the default then right?